frontend/model/contracts/voting/proposals.js

```diff
@@ -113,14 +117,7 @@ const proposals = {
   },
   [PROPOSAL_REMOVE_MEMBER]: {
     defaults: {
-      rule: RULE_PERCENTAGE,
-      expires_ms: 14 * DAYS_MILLIS,
-      ruleSettings: {
-        [RULE_PERCENTAGE]: { threshold: 0.8 },
-        // at least 2, since the member being removed shouldn't be able to
-        // block the proposal themselves
-        [RULE_DISAGREEMENT]: { threshold: 2 }
-      }
+      ...proposalDefaults
```

Comment:
Hmm, this line is different from the other ones... maybe it was done this way to take into account `[RULE_DISAGREEMENT]: { threshold: 2 }` ?

That was never re-incorporated here though. The comment also needs to be preserved.

Reply:
It should be the same, yes.

I didn't re-incorporated it because there are other types of "adjustments" needed based on other conditions. So, all adjustments are made at `rules.js` to keep everything together.